modules/install-consul/README.md

```diff
@@ -17,16 +17,15 @@ There is a good chance it will work on other flavors of Debian as well.
 
 <!-- TODO: update the clone URL to the final URL when this Module is released -->
 
-To install Consul, use `git` to clone this repository at a specific tag (see the [releases page](https://github.com/hashicorp/terraform-google-consul/tree/master/releases) 
-for all available tags) and run the `install-consul` script:
+To install Consul, use `git clone --branch` to clone this repository at a specific tag run the `install-consul` script:
 
 ```
-git clone --branch <VERSION> https://github.com/gruntwork-io/consul-gcp-module.git
-consul-gcp-module/modules/install-consul/install-consul --version 0.9.2
+git clone --branch <MODULE_VERSION> https://github.com/hashicorp/terraform-google-consul.git
+terraform-google-consul/modules/install-consul/install-consul --version 1.0.6
```

Comment:
Looks like Consul 1.10.0 was just released! Can you update to that version? On a separate note, we'll need to validate compatibility with Consul 1.10.0.